cdap-security-spi/src/test/java/co/cask/cdap/security/spi/authorization/AuthorizerTest.java

```diff
@@ -46,6 +49,10 @@ public void testSimple() throws UnauthorizedException {
     authorizer.grant(namespace, user, Collections.singleton(Action.READ));
     authorizer.enforce(namespace, user, Action.READ);
 
+    Set<Privilege> expectedPrivileges = new HashSet<>();
```

Comment:
shouldn't there be tests for role management in here - since the ``AuthorizationHttpHandlerTest`` always only uses ``InMemoryAuthorizer``? Or have you done this because you don't support RBAC in ``DatasetBasedAuthorizer``?

I think it would be worth it - primarily because we're providing a way to test all the functionality of ``Authorizer`` here, implementations just need to create the ``Authorizer`` and tests would be ready for them. You can suppress the tests for ``DatasetBasedAuthorizer`` right now, but we should really add tests here.